packages/frontend/src/pages/insight-editor/components/insight-editor-header/insight-editor-header.tsx

```diff
@@ -88,7 +88,7 @@ export const InsightEditorHeader = ({ insight, isPublishing, isSavingDraft, form
   const itemType = useWatch({
     control,
     name: 'itemType',
-    defaultValue: insight?.itemType
+    defaultValue: insight?.itemType || ''
```

Comment:
This and similar code below is for fixing previously imported insights that don't have `itemType` set. 

The issue is that `titleize()` expects a string and itemType is undefined for previous imported insights